paper-checked-element-behavior.html

```diff
@@ -51,6 +52,7 @@
   /** @polymerBehavior Polymer.PaperCheckedElementBehavior */
   Polymer.PaperCheckedElementBehavior = [
     Polymer.PaperInkyFocusBehavior,
+    Polymer.PaperRippleBehavior,
```

Comment:
`PaperInkyFocusBehavior` includes `PaperRippleBehavior`, so I don't see why this is needed https://github.com/PolymerElements/paper-behaviors/blob/master/paper-inky-focus-behavior.html#L48

Reply:
That's right, but when we bundle it, its not including the ripple behavior for some reason. It only works when I import it on checked element behavior

Reply:
What tool are you using to bundle? Is it handling transitive HTML import dependencies properly? Including `PaperRippleBehavior` twice is definitely not the way to go.

Reply:
I am using the polymer-cli's build

Reply:
@keanulee I am also running into errors with behaviors, when I refresh the page, it seems to disappear and give me a 404 behavior is null error. I had to add the behaviors into my main app element to get it to work.

Reply:
if you can provide some solutions, that would be great. How do I handle transitive HTML import dependencies?

Reply:
Can't repo this issue with polymer-cli - tried the following:

- Clone [polymer-starter-kit](https://github.com/Polymer/polymer-starter-kit/)
- Remove import for "paper-icon-button/paper-icon-button.html" from shop-app.html (https://github.com/Polymer/polymer-starter-kit/blob/141d2b5e1e6a2e58d544f40caded436b7325ecff/src/my-app.html#L22)
- `polymer build`
- `polymer serve build/es6-bundled` and open in browser. Observe `Polymer.PaperRippleBehavior` is not defined in console.
- Add import for "paper-behaviors/paper-checked-element-behavior.html" in shop-app.html
- `polymer build`
- `polymer serve build/es6-bundled` and open in browser. Observe `Polymer.PaperRippleBehavior` is now defined in console. (Don't forget to clear SW/shift+reload)